lib/addons/ack/service.ts

```diff
@@ -0,0 +1,348 @@
+import { AckAddOnProps, AckAddOn } from ".";
+import { supportsALL, supportsX86 } from "../../utils";
+import { AckServiceName } from "./serviceMappings";
+
+export interface ServiceAckAddOnProps
+    extends Omit<AckAddOnProps, "serviceName"> { }
+
+@supportsALL
+export class ACMAckAddOn extends AckAddOn {
```

Comment:
I dont think we should use this design approach of one class extension per ACK chart, this will make the design difficult the expand.